### PR TITLE
Add documentation on the mobile minter

### PR DESCRIPTION
**Pull request by shantanubala**
This PR adds documentation on the Mobile Minter that was previously in this Notion doc:

https://www.notion.so/artblocks/Mobile-Minting-for-IRL-Activations-7c1ee708e80e4b6fb102a3f27786788f?pvs=4

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204366113329910

### CODE REVIEW

**Review by jakerockland — APPROVED** [review]
@shantanubala This LGTM, but do you want to be called out explicitly here as the staff onboarding person? 

I think may be better to route these requests through @ABDruid / the BD team (even though they will end up getting routed to us, keeps this documentation a bit more generalized perhaps)?

**Comment by jakerockland** [comment]
@ABDruid @cooper-temple could y'all PTAL at the documentation here and add any additional thoughts/feedback/suggestions (doesn't need to block merging IMO, can iteratively adjust these based on feedback). 🍻 🙏

File: art-blocks-engine-onboarding/art-blocks-engine-101/Mobile-Minter.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+- Minting tokens without requiring manual signing and submission of Ethereum transactions
+
+### Requirements
+
+The Mobile Minter is perfect for situations where:
+
+- A project is paused and not available for online minting
+- An artist wants to distribute their work at a live event
+- An iPad or iPhone with the latest iOS version is available
+- A small amount of Ethereum is on hand to cover gas fees for token recipients
```
Comment (ABDruid) [inline art-blocks-engine-onboarding/art-blocks-engine-101/Mobile-Minter.md:25]: - You've set up a Stripe account to accept fiat payment

File: art-blocks-engine-onboarding/art-blocks-engine-101/Mobile-Minter.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+### What currencies are supported for payments in the Mobile Minter app?
+
+The Mobile Minter app processes fiat payments via Stripe Terminal, which supports a variety of currencies. The available currencies depend on your Stripe account and the country where your business operates. You will be the merchant of record, and Art Blocks will collect its platform fees in ETH or USD. For a list of supported currencies for your customers, please refer to the [Stripe documentation](https://stripe.com/docs/currencies).
+
+### Is it possible to customize the appearance of the Mobile Minter app for my event?
+
+While the Mobile Minter app does not offer customization, you can embed [live.artblocks.io](https://live.artblocks.io/) inside of a page that showcases your event's theme and branding. Please refer to the [Art Blocks documentation site](https://docs.artblocks.io/creator-docs/art-blocks-api/artblocks-viewer/#artblocks-viewer) for an overview of configuration options for the live viewer.
+
+### I have unspent ETH in the Mobile Minter's hot wallet. How can I retrieve the funds?
```
Comment (ABDruid) [inline art-blocks-engine-onboarding/art-blocks-engine-101/Mobile-Minter.md:137]: this entry can be combined with "Can my prepaid gas fee balance be returned to me?"

**Review by ABDruid — DISMISSED** [review]
added comments on two sections, but this is great! Thanks for putting this together

**Comment by jakerockland** [comment]
@shantanubala gentle ping on the minor adjustments suggested here and then I think this is merge ready

**Pull request merged**